apps/server/src/infra/schulconnex-client/response/schulconnex-laufzeit-response.ts

```diff
@@ -0,0 +1,19 @@
+import { IsDate, IsOptional, IsString } from 'class-validator';
+
+export class SchulconnexLaufzeitResponse {
+	@IsOptional()
+	@IsDate()
+	von?: Date;
+
+	@IsOptional()
+	@IsDate()
+	bis?: Date;
+
+	@IsOptional()
+	@IsString()
+	vonlernperiode?: string;
+
+	@IsOptional()
+	@IsString()
+	bislernperiode?: string;
```

Comment:
It is possible to have a response with "von" and with "vonlernperiode" and "bislernperiode". 

`... "gruppe": {
						"id": "bc1036fb-9767-4ef0-8584-2a6c4941d954",
						"bezeichnung": "Gruppe_Test_X",
						"typ": "Klasse",
						"laufzeit": {
							"von": "2022-08-31",
							"vonlernperiode": "2022-1",
							"bislernperiode": "2024-2"
						},
						"organisation": {
							"id": "a9a7d3f4-d4b0-4804-bf15-275637cb3bee",
							"kennung": "NI_00100",
							"name": "Testschulebfemaxypml_new",
							"typ": "Schule",
							"anschrift": {
								"ort": "Hannover"
							}
						}
					}, ....`

At the moment the login fails with an validation error "property personenkontexte[0].gruppen[1].gruppe.laufzeit.von has failed the following constraints: von must be a Date instance".

You can use brando wolf and "Gruppe_Test_X"
![image](https://github.com/hpi-schul-cloud/schulcloud-server/assets/12628806/0557ea34-8e0a-45bf-8b08-c92224f5c633)